modules/ROOT/assets/overlays/admin.yaml

```diff
@@ -0,0 +1,25 @@
+overlay: 1.0.0
+
+info:
+  title: Overlay for Admin REST API spec
+  version: 1.0.0
+
+extends: https://raw.githubusercontent.com/couchbase/sync_gateway/refs/heads/main/docs/api/admin.yaml
```

Comment:
This gets post processed, can this use ../attachments/bundled-admin.yaml instead? This is also fixed to main which isn't necessarily going to be right. Right now main is 3.3 release.

We could also just put some changes into sync_gateway repo for description and then it only has external_docs. This might be more relevant for capella, not sure.

Reply:
The `extends` key is just a mnemonic for users to show which file the overlay is extending, it doesn't actually do anything. The file which the overlay extends is selected by the `openapi-overlays-js` in [generate_docs.sh](https://github.com/couchbase/docs-sync-gateway/pull/839/files#diff-327361a918a7861cbe69407fe6a73895fa27d3f5cc86d03c5b9e7183f458668b).

I can see that this line is confusing though so I'll either update it or just delete it.

Reply:
Done